ZuluContent/Zulu/Mobiles/CreatureProperties.cs

```diff
@@ -94,73 +31,72 @@ public static CreatureProperties Get(Type T)
             return CreatureMap.GetValueOrDefault(T);
         }
 
-        public bool InitialInnocent { get; init; }
-        public CreatureProp Str { get; init; } = 0;
-        public CreatureProp Dex { get; init; } = 0;
-        public CreatureProp Int { get; init; } = 0;
-        public CreatureProp HitsMax { get; init; } = 0;
-        public CreatureProp DamageMin { get; init; } = 0;
-        public CreatureProp DamageMax { get; init; } = 0;
-        public CreatureProp ManaMaxSeed { get; init; } = 0;
-        public CreatureProp StamMaxSeed { get; init; } = 0;
-        public bool HasBreath { get; init; } = false;
-        public bool HasWebs { get; init; } = false;
-        public Poison HitPoison { get; init; } = null;
-        public bool CanSwim { get; init; } = false;
-        public bool CanFly { get; init; } = false;
-        public string Title { get; init; }
+        public string Name { get; init; } = "<CreatureProperties unset>";
         public string CorpseNameOverride { get; init; }
-        public string Name { get; init; } = "<MobileInitProperties unset>";
-        public Body Body { get; init; } = 0;
-        public Race Race { get; init; }
+        public Type BaseType { get; set; } = typeof(BaseCreatureTemplate);
+        public CreatureProp Str { get; init; }
+        public CreatureProp Int { get; init; }
+        public CreatureProp Dex { get; init; }
+        public CreatureProp ActiveSpeed { get; init; }
+        public AIType AiType { get; init; }
+        public bool? AlwaysAttackable { get; init; }
+        public bool? AlwaysMurderer { get; init; }
+        public bool? AutoDispel { get; init; }
+        public bool? BardImmune { get; init; }
+        public int? BaseSoundID { get; init; }
+        public Body Body { get; init; }
+        public bool? CanFly { get; init; }
+        public bool? CanRummageCorpses { get; init; } = true;
```

Comment:
Should this default to true?

Reply:
Yeah I think we might need a defaults file so they're not hardcoded and so it's easy on the server operators